.github/workflows/test.yml

```diff
@@ -58,6 +58,15 @@ jobs:
         with:
           path: ${{ env.BAZEL_CACHE_DIR }}
           key: ${{ runner.os }}-bazel-${{ env.CACHE_NUMBER }}
-      - name: Run tests
-        working-directory: tests/miniconda
+      - name: "Test miniconda w/ clean set to True"
```

Comment:
I've addressed all the comments, but the tests take a long time to run now. MacOS is the long tent in the pole which takes 20 minutes for its tests to complete.

Reply:
Let's just not clean cache in tests as this cache contains only downloaded tarballs and shouldn't be unsafe in any scenario. So only two tests are necessary `tests/miniconda` and `tests/mamba`. In fact there is another pull request with `miniforge` support (#21 ) orthogonal to using `mamba` so I would rather even name them `tests/miniconda/conda` and `tests/miniconda/mamba`. Later there will be added `tests/miniforge/conda` and `tests/miniforge/mamba`.

Reply:
Ok, I have done that. It looks like this now:

```
.
├── miniconda
│   ├── conda
│   └── mamba
└── miniforge
    ├── conda
    └── mamba
```

Reply:
Awesome